napari/view_layers.py

```diff
@@ -202,8 +199,8 @@ def view_multichannel(
     viewer.add_multichannel(
         data=data,
         axis=-1,
-        colormap=cmap,
-        contrast_limits=clims,
+        colormap=colormap,
```

Comment:
good catches, as noted in #632 this code will be deleted, but might as well fix as long as it is still there